modules/caddyhttp/celmatcher.go

```diff
@@ -473,8 +473,8 @@ func CELMatcherRuntimeFunction(funcName string, fac CELMatcherFactory) functions
 // The arguments are collected into a single list argument the following
 // function call returned: <funcName>(request, [args])
 func celMatcherStringListMacroExpander(funcName string) parser.MacroExpander {
-	return func(eh parser.ExprHelper, target *exprpb.Expr, args []*exprpb.Expr) (*exprpb.Expr, *common.Error) {
-		matchArgs := []*exprpb.Expr{}
+	return func(eh parser.ExprHelper, target ast.Expr, args []ast.Expr) (ast.Expr, *common.Error) {
```

Comment:
For the type here, prefer `cel.MacroExprFactory` and for the method signature, prefer `cel.MacroFactory`